var/spack/repos/builtin/packages/py-msgpack/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyMsgpack(PythonPackage):
+    """MessagePack (de)serializer."""
+
+    homepage = "https://msgpack.org/"
+    url      = "https://pypi.io/packages/source/m/msgpack/msgpack-0.6.2.tar.gz"
+
+    version('0.6.2', sha256='ea3c2f859346fcd55fc46e96885301d9c2f7a36d453f5d8f2967840efa1e1830')
+
+    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-cython@0.16:', type=('build', 'run'))
```

Comment:
Is cython required for the tarball or only for git checkouts?

Reply:
just double checked that it installs without cython. So it seems it is only necessary for when building from git checkout.